website/cue/reference/components/transforms/pipelines.cue

```diff
@@ -44,9 +44,12 @@ components: transforms: pipelines: {
 			warnings: []
 			type: object: options: {
 				order: {
-					common:      true
-					description: "A complete ordered list of how the pipelines will be chained."
-					required:    false
+					common: true
+					description: """
+						A complete ordered list of how the pipelines will be chained.
+						This field won't be used if the `parallel` mode is enabled.
```

Comment:
```suggestion
						A complete ordered list of how the pipelines will be chained.
						This field won't be used if the `route` mode is enabled.
```

However, this parameter will be used with `route` when we add support for routing only to the first matching pipeline.